src/app/components/pages/settings/blockchain/blockchain.component.ts

```diff
@@ -15,25 +15,28 @@ export class BlockchainComponent implements OnInit, OnDestroy {
   coinSupply: any;
   currentCoin: BaseCoin;
 
-  private coinSubscription: Subscription;
+  private coinSubscription: ISubscription;
 
   constructor(
     private blockchainService: BlockchainService,
     private coinService: CoinService
   ) { }
 
   ngOnInit() {
-    Observable.forkJoin(
-      this.blockchainService.lastBlock(),
-      this.blockchainService.coinSupply()
-    )
-    .subscribe(([block, coinSupply]) => {
-      this.block = block;
-      this.coinSupply = coinSupply;
-    });
-
     this.coinSubscription = this.coinService.currentCoin
-      .subscribe((coin: BaseCoin) => this.currentCoin = coin);
```

Comment:
`this.currentCoin = coin` is necessary. As it was deleted, now the box on the right does not appear.

Reply:
Fixed